cumulus/parachains/integration-tests/emulated/tests/bridges/bridge-hub-rococo/src/tests/snowbridge.rs

```diff
@@ -493,37 +494,6 @@ fn send_weth_asset_from_asset_hub_to_ethereum() {
 	});
 }
 
-#[test]
-fn register_weth_token_in_asset_hub_fail_for_insufficient_fee() {
```

Comment:
Why remove this test?

Reply:
https://github.com/Snowfork/polkadot-sdk/pull/125/commits/1d29ccd63387ee1f2bb471b54e9bbf407ad519e5 though I'd prefer to generate inbound fixtures only for basic flow and the use low-level functions(i.e. convert_xcm|send_xcm) for these edge cases.